lib/cli/package.json

```diff
@@ -20,7 +20,7 @@
     "dist": "rm -rf ../../dist/libs/cli && npm run build && cp -R ./bin ../../dist/libs/cli && cp -R ./resources ../../dist/libs/cli && cp -R ./templates ../../dist/libs/cli && cp ./package.json ../../dist/libs/cli"
   },
   "dependencies": {
-    "@alfresco/js-api": "^6.1.0",
+    "@alfresco/js-api": "6.2.0-982",
```

Comment:
should have caret otherwise may fail again

Reply:
Added